pom.xml

```diff
@@ -230,5 +230,15 @@
         <coveralls.skip>true</coveralls.skip>
       </properties>
     </profile>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>17</jdk>
+      </activation>
+      <properties>
+        <!-- coverall 4.3.0 or previous version does not work on Java 11, see https://github.com/trautonen/coveralls-maven-plugin/issues/112 -->
```

Comment:
- [ ] Java 11? The comments seem to be wrong.

  By the way, can I use it if I upgrade?
  However, regarding this one, we are not currently using the coveralls for various reasons, so we will leave it as a separate issue.

Reply:
Comment has been corrected.
We are not currently using coverall here, so we have not created an issue.
9fd678d583bcba63d319042a2943db9641a3d4e9